lisa/sut_orchestrator/azure/features.py

```diff
@@ -676,7 +676,7 @@ def get_all_disks(self) -> List[str]:
         raw_pattern = re.compile(r"(/sys/block/sd*[\w\W]*?)(?=/sys/block/sd*|\Z)", re.M)
         # sda
         # sdb
-        disk_pattern = re.compile(r"/sys/block/(?P<label>sd.*)", re.M)
+        disk_pattern = re.compile(r"/sys/block/(?P<label>sd*\w*)", re.M)
```

Comment:
Sorry, fixed a typo: `/sys/block/(?P<label>sd\w*)`
And I think it doesn't need both of them now. One pattern is enough, which can parse from the raw string directly.